docs/collections/graphs/using-rest-api.md

```diff
@@ -60,193 +60,196 @@ To create `edge collection` use same endpoint `/_fabric/{fabric_name}/_api/colle
-            'firstname': 'Jean',
-            'lastname':'Picard',
-            'email':'jean.picard@macrometa.io'
-        },
-        {
-            '_key':'James',
-            'firstname': 'James',
-            'lastname':'Kirk',
-            'email':'james.kirk@macrometa.io'
-        },
-        {
-            '_key': 'Han',
-            'firstname': 'Han',
-            'lastname':'Solo',
-            'email':'han.solo@macrometa.io'
-        },
-        {
-            '_key': 'Bruce',
-            'firstname': 'Bruce',
-            'lastname':'Wayne',
-            'email':'bruce.wayne@macrometa.io'
-        }
-    ]
-
-    url = FED_URL + "/_api/document/" + COLLECTION_NAME_1
-    resp = session.post(url,data=json.dumps(payload))
-    result = json.loads(resp.text)
-    print("\nDocuments Inserted: ",result)
-
-    payload = [
-        {'_id': 'lectures/CSC101', 'difficulty': 'easy', '_key':'CSC101', 'firstname':'Jean'},
-        {'_id': 'lectures/CSC102', 'difficulty': 'hard', '_key':'CSC102','firstname':'Jean'},
-        {'_id': 'lectures/CSC103', 'difficulty': 'hard', '_key':'CSC103','firstname':'Jean'},
-        {'_id': 'lectures/CSC104', 'difficulty': 'moderate', '_key':'CSC104','firstname':'Jean'}
-
-    ]
-
-    url = FED_URL + "/_api/document/" + COLLECTION_NAME_2
-    resp = session.post(url,data=json.dumps(payload))
-    result = json.loads(resp.text)
-    print("\nDocuments Inserted: ",result)
-
-    # Create Edge Collection
-
-    payload = { 'name': EDGE_COLL_NAME, "type":3 }
-
-    url = FED_URL + "/_api/collection"
-    resp = session.post(url,data=json.dumps(payload))
-    result = json.loads(resp.text)
-    print("\nEdge Collection Created: ",result)
-    payload = [
-        {
-        '_key': 'Jean-CSC101',
-        '_from': 'teachers/Jean',
-        '_to': 'lectures/CSC101',
-        'online': False
-        },
-        {
-        '_key': 'Jean-CSC102',
-        '_from': 'teachers/Jean',
-        '_to': 'lectures/CSC102',
-        'online': True
-        },
-        {
-        '_key': 'Jean-CSC103',
-        '_from': 'teachers/Jean',
-        '_to': 'lectures/CSC103',
-        'online': False
-        },
-        {
-        '_key': 'Bruce-CSC101',
-        '_from': 'teachers/Bruce',
-        '_to': 'lectures/CSC101',
-        'online': True
-        }
+""" This file is a demo on using edge collections and graphs """
+import json
+import requests
+
+# Constants
+
+FEDERATION = "api-gdn.paas.macrometa.io"
+FED_URL = f"https://{FEDERATION}"
+GEO_FABRIC = "_system"
+API_KEY = "my IP key" # Change to your API key
+#EMAIL = "my email"
+#PASSWORD = "my password"
+#AUTH_TOKEN = "bearer " # Do not change
+
+COLLECTION_NAME_1 = "teachers"
+COLLECTION_NAME_2 = "lectures"
+EDGE_COLL_NAME = "teach"
+GRAPH_NAME = "lectureteacher"
+
+# Create a HTTPS Session Part 1
+# The following is required when accessing Macrometa with your email and password instead of an API key or JWT token
+"""""
+url = f"{FED_URL}/_open/auth"
```

Comment:
Oops I might have missed this while reviewing the audit code... but we don't need to call /_open/auth endpoint here since we are using Api-key now, auth endpoint is used to get JWT token by logging in from email and password. Really sorry for the late comment.

Reply:
Hi Koshy.

Yes, I left so a user can use both possibilities, either an API key or email/pass. The latter is behind a multi-line comment, beginning with the URL variable... mmm constant. I'll check for PEP8 compliance on this part of the code.

Reply:
<img width="320" alt="image" src="https://user-images.githubusercontent.com/107413064/185224215-0d2cf82f-2828-472b-9f11-4e7f7fad8a90.png">
I guess you added 2 extra double quotes so I guess its not a comment anymore?

Reply:
OK, that part of the code has been updated.
Got a couple of unneeded double quotes.

Thanks for your review Koshy.